services/auth-service/package.json

```diff
@@ -2,16 +2,16 @@
   "name": "auth-service",
   "version": "1.0.0",
   "main": "index.js",
-  "type": "module",
   "dependencies": {
     "express": "^4.18.2",
-    "typescript": "^4.8.4"
+    "typescript": "^4.8.4",
+    "ts-jest": "^27.1.2"
   },
   "scripts": {
     "start": "node ./build/index.js",
     "build": "tsc",
     "dev": "nodemon",
-    "test": "tsc --noEmit"
+    "test": "jest"
```

Comment:
Maybe change this to `tsc --noEmit && jest` such that type-checking is a part of the pipeline test?

Reply:
Sure! I didn't know there was a difference

Reply:
> Maybe change this to `tsc --noEmit && jest` such that type-checking is a part of the pipeline test?

Sure!